modules/ROOT/pages/http/http-listener-ref.adoc

```diff
@@ -1,182 +1,296 @@
-= HTTP Listener Configuration Reference
+= HTTP Listener Reference
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint studio, esb, connectors, http, https, http headers, query parameters, rest, raml
 
-The required minimum settings for the Listener operation are:
+The HTTP listener is a source that allows setting up an HTTP server and triggering flows when HTTP requests are received.
 
-* Host
-* Port
-* Path
+You can choose what methods the source accepts, such as GET, POST or a list with both, and on which path requests will be accepted on, therefore allowing routing the requests through different flows.
```

Comment:
"or a list of them" instead of a "or a list of both". Otherwise it seems that only GET and POST are supported.

Reply:
what about the other verbs? the term `both` seams to imply that no other verb is supported.